compiler/scenario-service/server/src/main/scala/com/digitalasset/daml/lf/scenario/Conversions.scala

```diff
@@ -401,7 +401,7 @@ case class Conversions(homePackageId: Ref.PackageId) {
       case ex: N.NodeExercises[
             Ledger.NodeId,
             V.AbsoluteContractId,
-            V.VersionedValue[V.AbsoluteContractId]] =>
+            Tx.Value[V.AbsoluteContractId]] =>
```

Comment:
This part of the code doesn't quite seem to entail the "transaction value-ness" of most other code in this PR, so I've left it a little more independent than you might expect given `WithTxValue`, because I'd expect it to vary more easily.